devel/demo.html

```diff
@@ -51,6 +51,22 @@
       font-family: monospace;
       white-space: pre-wrap;
     }
+
+    #editor .stem-creator-container{
+      margin-bottom:100px;
+    }
+    .stem-creator-button{
+      background:red;
+      position:absolute;
+      width:30px;
+      height:30px;
+      border-radius:100px;
+      line-height: 30px;
+      text-align: center;
+    }
+    .stem-creator-button .symbol{
+      color:#fff;
+    }
```

Comment:
can we move some sort of minimal amount of css into the editor - as per your suggestion